src/olympia/ratings/migrations/0011_remove_rating_one_review_per_user_and_more.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 4.2.7 on 2023-11-15 15:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ratings', '0010_auto_20231106_1533'),
+    ]
+
+    operations = [
+        migrations.RemoveConstraint(
+            model_name='rating',
+            name='one_review_per_user',
+        ),
+        migrations.AlterField(
+            model_name='rating',
+            name='deleted',
+            field=models.IntegerField(default=0),
+        ),
+        migrations.AddConstraint(
+            model_name='rating',
+            constraint=models.UniqueConstraint(fields=('version', 'user', 'reply_to', 'deleted'), name='one_review_per_user'),
```

Comment:
not this pr/issue, but I think we either need to make `version` non-nullable or fix https://github.com/mozilla/addons-server/issues/10416 and drop version from ratings entirely, because null versions make this constraint pointless.  (Although it only works for replies in any case because null values again)

Reply:
I'm all for it - IIRC the problem is we have data to clean up (duplicate ratings) before we can do that, but we should bite the bullet and do it at some point.